docs/trigger-specs/CuriousTriggers.yaml

```diff
@@ -29,7 +29,7 @@ paths:
                   type: string
                   enum:
                     - EDUCATION_STARTED
-                    - ALL_EDUCATION_STOPPED
```

Comment:
change asked for by MN  - to be honest this file can probably be removed shortly.